samples/stream_load/java/DorisStreamLoad.java

```diff
@@ -95,6 +125,7 @@ protected boolean isRedirectable(String method) {
             StringEntity entity = new StringEntity(content);
             put.setHeader(HttpHeaders.EXPECT, "100-continue");
             put.setHeader(HttpHeaders.AUTHORIZATION, basicAuthHeader(DORIS_USER, DORIS_PASSWORD));
+            put.setHeader("label", "39c25a5c-7000-496e-a98e-348a264c81de")
```

Comment:
Would better add a comment for this line. the `label ` header is optional, not necessary.